src/extension.ts

```diff
@@ -8,12 +8,18 @@ export async function activate(context: vscode.ExtensionContext) {
   const smStore = new SourceMapStore();
   const includePattern = new ConfigValue("include", ["${workspaceFolder}"]);
   const excludePatterns = new ConfigValue("exclude", ["**/node_modules/**"]);
+
   const runner = new TestRunner(
     smStore,
     new ConfigValue("concurrency", 0),
     new ConfigValue("nodejsPath", "node"),
     context.extensionUri.fsPath,
-    new ConfigValue("nodejsParameters", [])
+    new ConfigValue("nodejsParameters", []),
+    new ConfigValue("extensions", [
```

Comment:
We'll want to treat the extensions similar to `includePattern` / `excludePatterns` since a change in them will require the controller to be re-created.

Reply:
Done,with onchange...